src/features/item-details/ItemProperties.scss

```diff
@@ -4,7 +4,7 @@
     margin: 0;
 
     dt {
-        color: $bdl-gray-62;
+        color: $bdl-gray-80;
```

Comment:
Just as a heads-up, it looks like this change affects a shared component that is used both in the Details sidebar within preview and in other areas:

- Have we validated the visual appearance for the other usages?
- Can/should we use a lighter gray for those areas if they have a `#fff` background, per @alexkrolick's suggestion? 

Same questions for the Classification component.

Reply:
As you can see in the screenshots provided, the color difference it's barely perceivable but for the Accessibility audit it makes a whole difference. We're on the clock with these adjustments, so I think it could be an alternative to generate a small technical debt regarding the special case for white BG, since its not something of significant impact rather than a 20% darker gray on the font color.

Is it viable to do this?

Reply:
We're going from `#909090` in Classification to `#4e4e4e`, so it's a pretty large color change in some places, to the point where the header isn't really distinguishable from the content. If the design team has signed off on the changes, though, the code itself LGTM.